specification/src/main/asciidoc/platform/Platform.adoc

```diff
@@ -23,6 +23,8 @@ include::ServiceProviderInterface.adoc[]
 
 include::CompatibilityMigration.adoc[]
 
+include::IntegrationRequirements.adoc[]
```

Comment:
Does there need to be a similar include for WebProfile.adoc and CoreProfile.adoc?

Reply:
Fixed in 6e4a0a21926f6cf60ff05e86fc372cbf9e1d3b42 .